src/client/store/pages/assessmentSection/stateType.ts

```diff
@@ -11,4 +11,6 @@ export type AssessmentSectionState = {
   nodeValueValidation: Record<string, NodeUpdate>
   // sectionName -> name -> content
   descriptions: Record<string, Record<string, string>>
+
+  dataLoaded: boolean
```

Comment:
this is specific to print view. I wouldn't add this property.
 in the hook useIsSectionDataEmpty can u verify that if some  data[countryIso][tableName] is undefined then data is not loaded ?

Reply:
Makes sense, updated